packages/node/src/utils/debug.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Has the debugger been enabled via the command line?
+ */
+export function isDebuggerEnabled(): boolean {
+  return process.execArgv.some(arg => arg.startsWith('--inspect'));
```

Comment:
Should we check for `inspector.url()` instead? https://nodejs.org/api/inspector.html#inspectorurl

Feels like a more robust check than the command line arg parsing.

Reply:
oh good point hmmm.

how do we go about combining the workers?